#!/usr/bin/env python

PACKAGE = "auv_physics"

import roslib; roslib.load_manifest(PACKAGE)

from driver_base.msg import SensorLevels
from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("force_neutral_buoyancy", bool_t, SensorLevels.RECONFIGURE_RUNNING, "Generate a force to cancel the buoyant force in the z-direction.", True)
gen.add("auto_start", bool_t, SensorLevels.RECONFIGURE_RUNNING, "Generate a force to cancel the buoyant force in the z-direction.", False)
gen.add("surface_level", double_t, SensorLevels.RECONFIGURE_RUNNING, "Position of the water surface", 0,    -50,  50)

exit(gen.generate(PACKAGE, "physics_simulator", "PhysicsSimulator" ))

